### PR TITLE
Add Nick Image

### PR DESCRIPTION
**Pull request by KSNINJA**
Added the Nick images to the Mentors and Students section
Included Screenshots of the changes in the mentors.yml and students.yml .
Added nick images to img/mentors and img/students

Link to the repo - https://github.com/KSNINJA/gci18.fossasia.org
Link to the hosted website. - https://ksninja.github.io/gci18.fossasia.org/

### CODE REVIEW

File: CNAME [hunk 1]
```diff
@@ -1 +0,0 @@
-gci18.fossasia.org
```
Comment (abishekvashok) [inline CNAME]: Please don't delete it please.

File: _data/students.yml [hunk 2]
```diff
@@ -11,4 +11,10 @@
 #   lng: Student's Longitude
 - name: Sahil Kumar
   github: sahildav
-  image: nick.png
\ No newline at end of file
+  image: nick.png
+
+
```
Comment (abishekvashok) [inline _data/students.yml]: Please remove these empty spaces.

**Comment by KSNINJA** [comment]
I have committed the changes
image
image

**Comment by KSNINJA** [comment]
But I have changed only 2 files - students.yml and mentors.yml
And I have also deleted the .DS_store file

**Comment by fragm3** [comment]
@KSNINJA this PR is for adding Nick Image, you have to add mentor in a separate issue(Task). Also, resolve the conflicts, squash your commits.

**Comment by KSNINJA** [comment]
Please guide me . I am getting confused . What should I do next ?
I have updated the mentors.yml and students.yml and added the nick images . I have also deleted the .DS_Store files

**Comment by simsausaurabh** [comment]
Try to squash commits and show clearly whatever changes you are making.

**Comment by tabesin** [comment]
Please add .DS_Store to your .gitignore

**Comment by KSNINJA** [comment]
This is the students.yml . 
image
This is the students.yml file . Even when I remove the red highlighted statement and other unwanted symbols. The same error is thrown and students.yml file automatically changes to this state again .

**Pull request merged**